assets/src/scripts/selectors/observations-selector.js

```diff
@@ -1,42 +1,43 @@
 import {extent} from 'd3-array';
 import {DateTime} from 'luxon';
 import {createSelector} from 'reselect';
-
-export const getObservationsCursorOffset = (state) => state.observationsState.cursorOffset || null;
-
 /*
- * Return a selector function which returns the current time series id or null if none define
- * @return {Function} - selector function which returns {String}
+ * Selectors to return observations state information or null if missing
  */
-export const getCurrentObservationsTimeSeriesId =
-    (state) => state.observationsState.currentTimeSeriesId ? state.observationsState.currentTimeSeriesId : null;
+export const getCurrentDVTimeSeriesId =
+    (state) => state.observationsState.currentDVTimeSeriesId ? state.observationsState.currentDVTimeSeriesId : null;
```

Comment:
Is this ternary really necessary or can `||` be used like the lines below?

Reply:
Necessary? No ... I happen to like the ternary syntax better. Is there a best practice around this?

Reply:
I don't think there's a best practice. I was just curious about between line switch away from the ternary.